MapboxSearch/ui/src/main/java/com/mapbox/search/ui/utils/extenstion/Context.kt

```diff
@@ -72,3 +83,27 @@ internal fun Context.resolveAttr(@AttrRes resId: Int): Int? {
         null
     }
 }
+
+internal fun Context.showToast(@StringRes resId: Int): Unit = Toast.makeText(this, resId, Toast.LENGTH_LONG).show()
```

Comment:
All these added functions should be unused. Aren't they?

Reply:
Removed